### ISSUE TITLE
Support threads

### ISSUE THREAD

**Issue by Phoenix616** [zachbr/Dis4IRC#58]
Discord is rolling out a new feature called "[Threads](https://support.discord.com/hc/en-us/articles/4403205878423-Threads-FAQ)". Simply forwarding new messages from them (maybe with some thread ID in the message?) should be enough imo.

I assume JDA needs to [add support](https://github.com/DV8FromTheWorld/JDA/pull/1613) first though :S

**Comment by TheCoolOne123** [comment]
JDA is working on this as told in the JDA Discord, but there is no idea how long this will take.

**Comment by zachbr** [comment]
Thanks - was trying to wrap my head around the best way to deal with this as well. My original thought was just to treat threads similarly to how we handle replies currently. So we'd just include the additional context of the previous message (n-1) with the latest message in that thread (n). A thread ID might work better but I'm not sure.

Honestly, bridging this sort of thing to IRC is never going to be ideal I suspect. If some guild makes a ton of threads under a channel then it also won't scale well. I guess we'll just have to try it and see once JDA adds support.

**Comment by TheCoolOne123** [comment]
Yeah, that's going to be a fun story @zachbr. JDA is working on this, but as told in their discord. For this to work, they have to rewrite the plugin to JDA v5 to support all the new features. And moving to discord api v9. So this is going to bring **A LOT** of deprication problems.

**Comment by Phoenix616** [comment]
Well the JDA v5-threads branch seems to have some recent activity which could enable this and Javacord now also supports it :eyes: (but I would hope JDA v5 addresses some of the issues that I see with the overall JDA design which made me choose Javacord instead so I doubt I need to push this project to another library too)
